Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Core/Src/Training.c

```diff
@@ -1,6 +1,63 @@
 #include "Training.h"
 
-void trainingTask(uint8_t* data)
+void trainingTask (uint8_t* data)
 {
+	validData = 0;
+	int motorOneState = motorCheck (*data);
+	int motorTwoState = motorCheck (*(data + 1));
+	int lightState = lightsCheck (*(data + 2));
+	
+	if (motorOneState && motorOneState == motorTwoState) // set outputArray values of motor if motors data are valid, motors are in sync
+	{
+		outputArray[0] = *data;
+		outputArray[1] = *(data + 1);
```

Comment:
Should write in array notation instead to be consistent and it is also what we use.

Reply:
What's the array notation